benchmarking/training_scripts/mlp_on_fashion_mnist/mlp_on_fashion_mnist.py

```diff
@@ -57,8 +63,8 @@
 
 
 _config_space = {
-    NUM_UNITS_1: randint(4, 1024),
-    NUM_UNITS_2: randint(4, 1024),
+    NUM_UNITS_1: lograndint(4, 1024),
```

Comment:
Made this change, because our own replacement logic maps `randint` to `lograndint` when `upper >= 100 * lower`.